data/human/deep jobs.txt

```diff
@@ -25,7 +25,16 @@ phrase "deep mystery cube pickup"
 	word
 		` `
 	word
-		`and pick up the mystery package. It is a small black cube, about two centimeters to a side, surprisingly heavy for its size, with no markings of any kind. Odd.`
+		`and pick up the mystery package. It is a small black cube, about two centimeters to a side, surprisingly heavy for its size, with no markings of any kind.`
+	word
+		` `
+	word
+		`Odd.`
+		`Strange.`
+		`Bizarre.`
```

Comment:
I'd remove this one as it's a stronger word than "odd" or "strange", and also including it reduced the player's chance of seeing any of the new messages you've added .

Reply:
I'd maybe replace it with e.g. `Curious.` / `How curious.`.

IMO it is fine that the new text only shows up half the time--that might even be too much.

We could also double-weight the remarks, e.g. 
```
`Odd.`
`Odd.`
`Strange.`
`Strange.`
<...new flavor warnings>
```
to have them be just a touch more common, in aggregate, than the warnings.